src/services/navigation/useNavLinks/useStaticNavLinks.js

```diff
@@ -46,6 +46,42 @@ export function useStaticNavLinks() {
       isExternalLink: true,
       openNewTab: true,
     },
+    termsOfService: {
```

Comment:
Did a little bit of reorganization here moving all the links with "MARKETING_BASE_URL" to the top